package.json

```diff
@@ -39,7 +39,7 @@
   "defaults": {
     "roslyn": "4.11.0-2.24267.3",
     "omniSharp": "1.39.11",
-    "razor": "7.0.0-preview.24266.1",
+    "razor": "9.0.0-preview.24311.4",
```

Comment:
Can you also delete the entries in the package.json below that reference the razor bits from the blob storage?

Reply:
oh yup! Missed that